lib/app/components/pages/search_filter/controllers/search_filter_controller.dart

```diff
@@ -2,54 +2,54 @@ import 'package:get/get.dart';
 import 'package:favoritism_communication/app/services/gender_service.dart';
 
 class SearchFilterController extends GetxController {
-  final Rx<Gender?> _gender = Rx<Gender?>(null);
-  final Rx<int?> _age = Rx<int?>(null);
+  final Rx<Map<Gender, bool>> _gender = Rx<Map<Gender, bool>>({});
 
```

Comment:
```suggestion
@override
  void onInit() {
    super.onInit();
    // 初期値で性別は全てTrueにする
    for (Gender gender in Gender.values) {
      changeGender(gender, true);
    }
  }
```
`SearchFilterView`の表示直前に書いていた処理をここに移動お願いします